site/app/entities/poll/Poll.php

```diff
@@ -58,16 +66,17 @@ class Poll {
     #[ORM\JoinColumn(name: "poll_id", referencedColumnName: "poll_id")]
     protected Collection $responses;
 
-    public function __construct(string $name, string $question, string $question_type, \DateTime $release_date, string $release_histogram, string $release_answer, string $image_path = null) {
+    public function __construct(string $name, string $question, string $question_type, \DateInterval $duration, \DateTime $release_date, string $release_histogram, string $release_answer, string $image_path = null, ) {
         $this->setName($name);
         $this->setQuestion($question);
         $this->setQuestionType($question_type);
+        $this->setDuration($duration);
+        $this->setEndDate(new \DateTime("1900-01-02"));
```

Comment:
Is there a reason for this default end date?

Reply:
It should 1900-01-01, I changed that.